docs/DeveloperGuide.adoc

```diff
@@ -12,7 +12,7 @@ ifdef::env-github[]
 :note-caption: :information_source:
 :warning-caption: :warning:
 endif::[]
-:repoURL: https://github.com/CS2113-AY1819S1-F09-2/main/tree/master
+:repoURL: https://github.com/CS2113-AY1819S1-F09-2/main/blob/master
```

Comment:
I made a mistake here, can you change the repoURL back to `https://github.com/CS2113-AY1819S1-F09-2/main/tree/master` ?